src/ducks/export/services.js

```diff
@@ -0,0 +1,163 @@
+import { Q } from 'cozy-client'
+import { format } from '@fast-csv/format'
+
+import { FILES_DOCTYPE, TRANSACTION_DOCTYPE } from 'doctypes'
+import categories from 'ducks/categories/tree'
+
+const DATE_FORMAT = 'YYYY-MM-DD'
+const ROOT_DIR_ID = 'io.cozy.files.root-dir'
+
+const dateStr = date =>
+  date != null ? formatDate(date, DATE_FORMAT) : undefined
```

Comment:
formatDate is not defined, right? 

nit: You tend to use strict equality (!==) in our front end app.

Reply:
Ah you're right, I missed the import line when committing.

I use strict equality unless I don't want to :). Here I check that `date` is not equal to `null` AND `undefined`. I wouldn't get that with strict equality.